src_web/kamaho-shokusu/src/Policy/ApprovalPolicy.php

```
@@ -23,43 +23,43 @@ public function __construct(?RoomAccessService $roomAccessService = null)
     }
 
     /** ブロック長用承認一覧 */
-    public function canBlockLeaderIndex(?IdentityInterface $user, mixed $resource): bool
+    public function canBlockLeaderIndex(?IdentityInterface $user, \App\Controller\ApprovalController $resource): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$resource'. (undefined)

(UnusedFormalParameter)